### ISSUE TITLE
Total biomass/numbers checks

### ISSUE THREAD

**Issue by mike-levine** [noaa-afsc-mace/WinterGOAHistorical#2]
Hey @abigailatsea ,

I am going to start using this package in the winter GOA reports, so figured I'd do some basic checks.

I basically just summed up three sources for the pre-selectivity corrected era:
- Your new package
- The dataset that is included in the current 2024 cruise report
- The tables from the 2023 SAFE report.

I put the results in:
G:\MikeLevine\testing\test_goadata_package\compare_sources.xlsx
There's a tab for 'by length' comparisons, and a tab for 'by age'. Things look really good overall! A few notes/questions
- Comparisons are % change from the report or the SAFE to the new package values. I highlighted them in green if <1%, yellow if a few %, red, if a lot %. The 'red' cases are all pretty clearly mistakes in the dataset that is in the report (the SAFE and the package data agree). So it will be good to clear those out and replace with better historical values! I think the package data is 'correct' and we should trust it, and it is really similar to the SAFE values.
- And a few requests, maybe:
 1. Is it possible to get the Shumagins 1994/1995/1996 values? These exist in the report dataset but not the package dataset.
 2. In `WinterGOAHistorical::PreSelectivityWinterGOAages`, the regions are a little funky- any reason that 1995, 1996, and 1997 are 'other'?
![image](https://github.com/user-attachments/assets/07ac702c-5c0a-40a9-9fe6-b75ecf895414)
3. When you queried these values, is transect width something that is easily added? I don't really know the structure of macebasehistoric, or how the queries are written. If its as easy as adding a word to the query, that would be great. If you'd have to do some complicated thing to assign widths based on vessel log, etc., it isn't worth it. 

Thanks, this seems really close, and really helpful!

**Comment by abigailatsea** [comment]
I had to put this dev aside to focus on the EBS, working to address all of the issues now.

**Comment by abigailatsea** [comment]
I updated the script to fetch historical ages so it matched with the one to get historical lengths, which fixed the region == 'other' issue. 
The Shumagins surveys only go back to 2001 (!!!) in MB2. We briefly discussed this, and IIRC decided to not try to resurrect the pre-2001 Shumagins surveys because they exist just as csvs in old cruise folders or on MB historic. 

Looking into Q3.

![shelikof_surveys_1995_2007](https://github.com/user-attachments/assets/0a4a0d70-a156-4bc5-aa23-221f47e07402)

**Comment by mike-levine** [comment]
Ok sorry for the delay, just getting back to this part of the cruise report.

In the version of the package I downloaded just before swapping to a private repo, could you check what data you've got? For some reason I only get two surveys now (199504 and 200102).  So for example in the old data package for the Shelikof I got:
![image](https://github.com/user-attachments/assets/fa8c5a62-afdb-4825-b167-dc6716e868e4)

and in the update:
![image](https://github.com/user-attachments/assets/70669038-ab58-425b-9b97-b8e73cb4ab20)

Thanks for checking @abigailatsea! It could just be wackiness related to the swap to private? I'm not sure.

**Comment by abigailatsea** [comment]
Yes, I saw that, realized my mistake (I compiled the data in test mode with just two surveys), and the full data set is in the new package. I'm going to make you an admin on the updated version and see if that helps with the install issue.

**Comment by mike-levine** [comment]
Cool- I think this might argue for going back to public mode! You probably don't want to have to control access to each person in the group? That's sure been an unanticipated side effect of the move.

**Comment by abigailatsea** [comment]
You know what? Let me try making it public again and seeing if it works properly!

**Comment by abigailatsea** [comment]
@mike-levine Can you try now? I made it public and re-published the website.

**Comment by mike-levine** [comment]
Yep I could install from that version no problem!

**Comment by abigailatsea** [comment]
Yay! Does it look ok?

**Comment by mike-levine** [comment]
I'll check things out next week more and get back to you on that, thanks for sorting out the install!

**Comment by mike-levine** [comment]
Cool all looks good! The only remaining question is this one, from way up the chain:
When you queried these values, is transect width something that is easily added? I don't really know the structure of macebasehistoric, or how the queries are written. If its as easy as adding a word to the query, that would be great. If you'd have to do some complicated thing to assign widths based on vessel log, etc., it isn't worth it.

**Comment by abigailatsea** [comment]
@mike-levine I have included transect width as requested, let me know if that does it for you!

**Comment by mike-levine** [comment]
This looks good to me, see anything amiss? All the areas calculated from the historical *Width* column look reasonable. 

![image](https://github.com/user-attachments/assets/0dce09ef-eda2-4e1d-a52f-a8693d760396)

**Comment by abigailatsea** [comment]
That looks reasonable to me, all I see is that the caption should be updated to "...surveys conducted since 1995". 
I have also not entirely given up on resurrecting the old-old shumagins surveys from the scraps of old csvs. I think it can be done, but it's a project that will take some time!
